src/utils/colors.js

```diff
@@ -33,13 +42,16 @@ export const getCustomThemeColors = ({ theme, customThemeValue }) => {
     button = theme === 'dark' ? dark.button : light.button;
     textarea = theme === 'dark' ? dark.textarea : light.textarea;
   }
+
   return {
     header,
     pane,
     body,
     composer,
     button,
     textarea,
+    susiBackgroundColor,
```

Comment:
Rename to `susiMessageBackgroundColor` and `userMessageBackgroundColor` everywhere.

Reply:
Changes have been Done